src/pymodaq_gui/managers/action_manager.py

```diff
@@ -51,7 +56,7 @@ def __repr__(self):
         return f'QAction {self.text()}'
 
 
-def addaction(name: str = '', icon_name: str = '', tip='', checkable=False, checked=False,
+def addaction(name: str = '', icon_name: Union[str,QtGui.QIcon] = '', tip = '', checkable=False, checked=False,
```

Comment:
it's not QtGui.QICon as we use the enum name from the Theme, therefore it's still only a string **or a Path**

Reply:
I initially thought it is way such that the user can still give their QIcon directly